functions/responsestreaming/Function.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_response_streaming]
+
+using Google.Cloud.Functions.Framework;
+using Microsoft.AspNetCore.Http;
+using System.Threading.Tasks;
+using Google.Cloud.BigQuery.V2;
+using System;
+
+namespace StreamBigQuery;
+
+public class Function : IHttpFunction
+{
+    public async Task HandleAsync(HttpContext context)
+    {
+        string projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        BigQueryClient client = BigQueryClient.Create(projectId).WithDefaultLocation(Locations.UnitedStates);
+        string query = @"
+            SELECT abstract FROM `bigquery-public-data.breathe.bioasq` LIMIT 10";
+        
+        BigQueryJob job = client.CreateQueryJob(
+            sql: query,
+            parameters: null,
+            options: new QueryOptions { UseQueryCache = false });
+        // Wait for the job to complete.
+        job = job.PollUntilCompleted().ThrowOnAnyError();
+        // Display the results
+        foreach (BigQueryRow row in client.GetQueryResults(job.Reference))
+        {
+            await context.Response.WriteAsync($"{row["abstract"]}" + "\n");
```

Comment:
```suggestion
            await context.Response.WriteAsync($"{row["abstract"]}\n");
```

(There's no point in using string interpolation at all if the string *only* contains an interpolated value - so *either `row["abstract"] + "\n"` would be okay, or the version I've given above - I'd prefer the version above.

Reply:
Thank you for reviewing! Addressed comments in latest commit